src/helperFunctions/getAssigneeProfilePic.ts

```
@@ -0,0 +1,14 @@
+import { USERS } from '@/components/constants/url';
+import fetch from '@/helperFunctions/fetch';
+
+const getAssigneeProfilePic = async (username: string) => {
+     try {
+          const url = `${USERS}/${username}`;
+          const { requestPromise } = fetch({ url });
+          const { data: result } = await requestPromise
+          return result.user?.picture?.url
+     } catch (err: any) {
```

Why error is not handled here?

---

done